BlobStorage/.editorconfig

```diff
@@ -12,4 +12,7 @@ charset = utf-8
 insert_final_newline = true
 
 [*.json]
-indent_size = 2
\ No newline at end of file
+indent_size = 2
+
+[Makefile]
+indent_style = tab
```

Comment:
We should include a new line at the end of each file. 

"The message is displayed because otherwise there is no way to tell the difference between a file where there is a newline at the end and one where is not. Diff has to output a newline anyway, or the result would be harder to read or process automatically."

Reply:
Added